documents/tinyengine/advanced-features/using-utils-methods.md

```
@@ -0,0 +1,169 @@
+# 工具类方法 utils
+
+## 基本概念
+
+在常规代码开发中，我们通常会将一些高频率用到的一些代码片段抽离出来业务代码，使其成为一个公共函数，减少重复的代码，从而达到代码复用的目的。
+同样，在低代码开发中，我们不可避免的需要编写一些高代码进行组合开发，这里同样存在存储一些公共函数以供各处调用需求。工具类方法 utils 就是诞生在这样的背景之下，我们可以将一些可复用的公共函数编写到工具类方法中，也可以将一些 npm 包引用到utils中，供后续调用。
+
+**注意：utils 是应用级别的，即引入之后，即可在该应用下任意页面中进行调用**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)